Gems/EMotionFX/Code/EMotionFX/Source/Mesh.cpp

```diff
@@ -1606,8 +1608,8 @@ namespace EMotionFX
     void Mesh::ConvertTo32BitColors()
```

Comment:
This looks like dead code that can be removed. At this point the EMFX VertexAttributeLayers are only needed for things that EMFX accesses on the CPU, so positions, id's, and weights for CPU skinning (used for ray-casting/hit detection in the editor), and tangents and bitangents (used for the EMFX editor visualization, which is skinned and then submitted from the CPU). But there is nothing on the CPU that should need vertex colors.

To actually visualize vertex colors in the EMFX editor, you would need to apply a material that outputs the vertex color. But those colors would come from the Atom mesh, not the EMFX mesh, so nothing should be needed in EMFX for vertex colors to come through.

Reply:
that simpler to do :+1:

Reply:
Great, thanks for confirming @amzn-tommy. So we can get rid of that whole layer